src/mumble/Settings.cpp

```diff
@@ -968,12 +968,24 @@ void Settings::legacyLoad(const QString &path) {
 	LOAD(iChatMessageMargins, "ui/ChatMessageMargins");
 	LOAD(bDisablePublicList, "ui/disablepubliclist");
 
-	bool bAskOnQuit = true;
-	LOAD(bAskOnQuit, "ui/askonquit");
-
 	if (settings_ptr->contains("ui/connectedquitbehaviour")) {
 		// Load the new connected quit option
-		LOADFLAG(connectedQuitBehaviour, "ui/connectedquitbehaviour");
+		LOADFLAG(connectedQuitBehavior, "ui/connectedquitbehaviour");
+	} else {
+#ifndef Q_OS_MACOS
```

Comment:
Why the guard against macOS?

Reply:
That is something I wonder about. I thought the guard is because this option does not exist in MAC. What is the reason for the guard in MainWindow::closeEvent for the asking dialog?

Reply:
The setting does exist - it just seems to not be used since https://github.com/mumble-voip/mumble/commit/c577d450056362487576c7de40a5e1a9a0c9a370
Unfortunately the commit does not give any indication whatsoever as to why it does what it does :shrug: 

But I'd argue that at least here in this code, we don't need the #ifdef